client/state/ui/nps-survey-notice/test/reducer.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { NPS_SURVEY_DIALOG_IS_SHOWING } from 'state/action-types';
+import reducer, { isNpsSurveyDialogShowing } from '../reducer';
+
+describe( 'reducer', () => {
+	it( 'should export expected reducer keys', () => {
+		expect( reducer( undefined, {} ) ).to.have.keys( [
+			'isNpsSurveyDialogShowing'
+		] );
+	} );
+
+	describe( '#isNpsSurveyDialogShowing()', () => {
+		it( 'should default to false', () => {
+			const state = isNpsSurveyDialogShowing( undefined, {} );
+
+			expect( state ).to.be.false;
+		} );
+
+		it( 'should track if the dialog is showing', () => {
+			const state = isNpsSurveyDialogShowing( undefined, {
+				type: NPS_SURVEY_DIALOG_IS_SHOWING,
+				isShowing: true
+			} );
+
+			expect( state ).to.be.true;
+		} );
```

Comment:
Is it worth adding a test for dismissing / setting `isShowing: false` here as well?